R/utilities-sensitivity-analysis.R

```diff
@@ -749,7 +809,9 @@ plotPopulationSensitivity <- function(structureSets,
             "Value" = missingSensivitity,
             "Quantile" = qu,
             "individualId" = id,

```

Comment:
Shouldn't that be IndividualId (capital I)

Reply:
Fixed